examples/customresources/models/mygizmo/mygizmo.go

```diff
@@ -15,8 +15,24 @@ import (
 // Model is the full model definition.
 var Model = resource.NewModel("acme", "demo", "mygizmo")
 
+// Config is the gizmo model's config.
+type Config struct {
+	Arg1 string `json:"arg1"`
+}
+
+// Validate checks the attribute of myGizmo to ensure that an "arg1" attribute exists
+// the model will not initialize if this is not set.
```

Comment:
```suggestion
// Validate ensures that `Arg1` is a non-empty string. Validation error will stop the associated resource from building.
```
"attributes" are not really a thing at this point in the code, and "model" is confusing (it's really a resource that does or does not get initialized).